src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/visits/PrisonVisitsService.kt

```diff
@@ -253,7 +251,6 @@ private val vsipToNomisOutcomeMap = mutableMapOf(
   VsipOutcomeStatus.PRISONER_REFUSED_TO_ATTEND to NomisCancellationOutcome.REFUSED,
   VsipOutcomeStatus.VISITOR_CANCELLED to NomisCancellationOutcome.VISCANC,
   VsipOutcomeStatus.VISIT_ORDER_CANCELLED to NomisCancellationOutcome.VO_CANCEL,
-  VsipOutcomeStatus.BATCH_CANCELLATION to NomisCancellationOutcome.BATCH_CANC,
-  VsipOutcomeStatus.ADMINISTRATIVE_CANCELLATION to NomisCancellationOutcome.ADMIN_CANCEL,
```

Comment:
This is a the bug, ADMINISTRATIVE_CANCELLATION was mapped twice, with the second one mapped incorrectly